tinygrad/tensor.py

```diff
@@ -132,6 +132,7 @@ def numpy(self) -> np.ndarray:
 
   def to(self, device:Optional[str]) -> Tensor:
     if device is None or device == self.device: return self
+    if device == "CPU" and self.lazydata.dtype.np is None: raise RuntimeError("{} buffer is not supported on CPU".format(self.lazydata.dtype))
```

Comment:
This doesn't belong here, there needs to be a generic way to register unsupported dtypes for a device.